src/Symfony/Component/Console/Formatter/OutputFormatterStyle.php

```diff
@@ -166,7 +166,7 @@ public function apply($text)
         $unsetCodes = [];
 
         if (null === $this->handlesHrefGracefully) {
-            $this->handlesHrefGracefully = 'JetBrains-JediTerm' !== getenv('TERMINAL_EMULATOR') && !getenv('KONSOLE_VERSION');
```

Comment:
if it doesn't work with older versions of use, shouldn't you check the version here, instead of enabling it for everyone?

Reply:
Right, good idea!